config/initializers/filter_parameter_logging.rb

```diff
@@ -3,6 +3,5 @@
 
 # Configure sensitive parameters which will be filtered from the log file.
 Rails.application.config.filter_parameters += [
-  :password,
```

Comment:
Would prefer a comment here to indicate these symbols are "includes" and not exact matches. (Basically something similar to the description of this PR.)

Reply:
That's reasonable. It's a weird behavior if you haven't seen it before.